lib/morse_translator.rb

```diff
@@ -40,4 +42,32 @@ def initialize
                     " " => " "}
   end
 
+  def translate(string)
+    morse_code = ""
+    string.downcase.split("").each do |letter|
+      morse_code << @dictionary[letter]
+    end
+    return morse_code
```

Comment:
What you are doing is creating a container, and then for each element, changing it somehow and putting it into the container. In the future, I want you to recognize this as a map. What you did looks great though!